docs/02-guides/magic-button/auto-refresh.mdx

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 3
+title: Auto Refresh
+description:
+  With this feature, your data is consistently updated every few seconds,
+  providing real-time updates for contributors.
+---
+
+import BrowserWindow from "@site/src/components/BrowserWindow";
+import SinceVersion from "@site/src/components/SinceVersion";
+
+<SinceVersion tag="2.25" />
```

Comment:
```suggestion
<SinceVersion tag="3.2" />
```

BTW, I wonder if it makes sense to add them given our current context and stage… It's maybe a bit early to document each new versions of 2.x features being ported because our users are early adopters up-to-date with our features.

But I don't have a strong opinion on this.

Reply:
Without more feedback, I've just fixed the version number but kept the `<SinceVersion />` component.